### PR TITLE
QPPA-3293, QPPA-3294, QPPA-3295 - Add 2020 Quality, PI, IA measures

### PR DESCRIPTION
**Pull request by chouinar**
https://jira.cms.gov/browse/QPPA-3293
https://jira.cms.gov/browse/QPPA-3294
https://jira.cms.gov/browse/QPPA-3295

#### What is being changed

This adds the 2020 measures for quality, PI and IA measures as well as makes a few adjustments to the scripts for consuming 2020 data, although these are largely the same as 2019s ingestion scripts. Also includes some minor updates for 2019 PI measures.

Detailed information about what is being changed and rationale for interesting decisions made

#### Release checklist:
Tasks that must be done prior to merging this PR, including testing.

* [x] [Package version updated](https://github.com/CMSgov/qpp-measures-data/blob/master/CONTRIBUTING.md#versioning-publishing-and-creating-new-releases)
* [x] Documentation updated
* [x] Unit tests added/passing
* [x] Verified working locally

### CODE REVIEW

File: util/measures/2019/pi-measures.csv [hunk 1]
```diff
@@ -1,40 +1,40 @@
-measureId,title,description,isRequired,metricType,firstPerformanceYear,weight,objective,isBonus,reportingCategory,substitutes,measureSpecification,exclusion
```
Comment (sheldon-b) [inline util/measures/2019/pi-measures.csv]: Why was this file changed?

**Comment by sheldon-b** [comment]
Oof, that's kind of a negative tone. Thank you for doing this 🙏 just curious about what sort of qa process we have in place, if any

File: util/measures/2019/pi-measures.csv (hunk 1 above)
Comment (chouinar) [reply util/measures/2019/pi-measures.csv]: There was an update for 2019 data as well, mostly just affecting a few descriptions.

**Comment by chouinar** [comment]
@sheldon-b  Scripts were largely unchanged from 2019. A few column names changed, a few new valid values existed in some fields. Some columns came in as something like "My Value" when they needed to be camelCased as "myValue" so I camel-case the column.

If it helps, I could do some sort of revert, copy the scripts from 2019->2020, and then put my changed versions in a separate commit so you can see the actual changes. Let me know if that would help you review.

As for tests, let me look if I should add some more. There is a test that checks that the 2020 measures is valid that is based on the config values, so it wasn't explicitly written.

**Comment by sheldon-b** [comment]
Okay, that's fine, thank you. I'm satisfied that _some_ tests are running. We can think more about tests in the future
